lib/process.js

```diff
@@ -34,7 +34,7 @@ function changePartsIfFunctional (parts) {
   if (isFunctionalTemplate(parts)) {
     parts.lang = 'javascript'
     const functionalProps = extractPropsFromFunctionalTemplate(parts.template.content)
-    parts.template.content = parts.template.content.replace('props.', '')
+    parts.template.content = parts.template.content.replace(/props./g, '')
```

Comment:
Can you explain why you remove props from the template?

Reply:
I know this code is a bit cryptic. What I basically do is detecting all prop definitions through a regular expression and then I remove the "props." prefix cause it won't be present when parent templates pass properties (you can see how this works in process-functional.spec.js that will be renamed to extract-props.spec.js)